packages/api/src/wallets/DID.ts

```diff
@@ -106,21 +107,63 @@ export default class DIDWallet extends Wallet {
     }>('did_get_current_coin_info', args);
   }
 
-  async getDidInfo(args: { coinOrDIDId: string }) {
+  async getDidInfo(args: { coinId: string; latest?: boolean }) {
```

Comment:
use getDIDInfo  instead of getDidInfo.
now it is inconsistent

Reply:
Oh, I thought I beated these inconsistent word issues but still some of them still remain...
I'll open a new PR which addresses this issue!